requirements-dev.txt

```diff
@@ -1,8 +1,11 @@
+cython
 flake8
 flake8-print
 notebook
 pytest
 pytest-cov
+pytest-cython
 restructuredtext-lint
 sphinx
 sphinx_rtd_theme
+tqdm
```

Comment:
So, if someone calls hyperparamopt and there is no progress, one has to check the log file as there will be nothing on the console? I hope there are easy ways to toggle this switch (i.e. set log file to stdout)